prime-angular-frontend/src/app/shared/components/forms/unlisted-college-licence-form/unlisted-college-licence-form.component.html

```diff
@@ -0,0 +1,58 @@
+
+<ng-container [formGroup]="form">
+
+  <app-form-icon-group [show]="total >= 1"
+    (event)="removeUnlistedCertification()">
+    <div class="row">
+      <div class="col">
+
+        <mat-form-field class="w-100">
+          <input matInput
+                 placeholder="College Name"
+                 formControlName="unlistedCollegeName"
+                 maxlength="100">
+          <mat-error *ngIf="unlistedCollegeName.errors?.['required']">Required</mat-error>
+        </mat-form-field>
+
+      </div>
+    </div>
+    <div class="row">
+      <div class="col">
+
+        <mat-form-field class="w-100">
+          <input matInput
+                 placeholder="College Code"
+                 formControlName="unlistedCollegeCode"
+                 maxlength="30">
```

Comment:
A few things, (1) got this error when provided "E5" as input 

![image](https://user-images.githubusercontent.com/7586665/179899743-4f85e876-4da6-4c67-bf2b-e029309c4b12.png)

(2) The mock-up names this field differently:  https://xd.adobe.com/view/2c575e9f-46f3-4c99-a0e6-89be106407a6-7904/screen/8790f734-8454-4eeb-9692-70677e95f65e/ 

(3) Please check the mock-up for length limitations for the fields